plugin/src/main/java/com/denizenscript/denizen/events/player/PlayerSteersEntityScriptEvent.java

```diff
@@ -38,6 +40,8 @@ public class PlayerSteersEntityScriptEvent extends BukkitScriptEvent {
     //
     // @Player Always.
     //
+    // @deprecated Use the 'player input' on versions above 1.21.

```

Comment:
Minor typo here (`Use the 'player input' ?? on versions`), and also I believe we usually do e.g. `deprecated on MC 1.21+` if you look at other [recent meta](https://meta.denizenscript.com/Docs/Tags#entitytag.knockback), otherwise LGTM!
Does have a merge conflict though, feel free to ask on Discord if you need any help with that.